config/dev-realm-spsh.json

```diff
@@ -1237,6 +1238,60 @@
           "offline_access",
           "microprofile-jwt"
         ]
+      },
+      {
+        "id": "8d129be5-0cde-44e7-b9de-59797f650e4b",
+        "clientId": "${KC_NEXTCLOUD_CLIENT_ID}",
+        "name": "NextCloud-Dateiablage",
+        "description": "",
+        "rootUrl": "",
+        "adminUrl": "",
+        "baseUrl": "",
+        "surrogateAuthRequired": false,
+        "enabled": true,
+        "alwaysDisplayInConsole": false,
+        "clientAuthenticatorType": "client-secret",
+        "secret": "${KC_NEXTCLOUD_CLIENT_SECRET}",
+        "redirectUris": [
+          "https://sh.lumaserv.dev/*"
+        ],
```

Comment:
Usually you'd set the `rootUrl` to the URL (i.e. `https://sh.lumaserv.dev`) and set `redirectUris` to relative paths (in this case `/*`. Speaking of which, can you narrow this down further instead of using the wildcard?)